AD7173.cpp

```diff
@@ -275,3 +275,23 @@ int AD7173Class::set_offset_config(adc7173_register_t offset, uint32_t offset_va
 	/* return error code */
 	return 0;
 }
+
+
```

Comment:
Have consistency with rest of the code. Remove this extra line

Reply:
Sorry for that issue. I corrected it and squashed.